packages/shell-api/src/shard.spec.ts

```diff
@@ -2516,6 +2518,26 @@ describe('Shard', function () {
         );
       });
     });
+    describe('automerge', function () {
+      it('not shown if sh.status() if not explicitly enabled', async function () {
+        expect((await sh.status()).value.automerge).is.undefined;
```

Comment:
this works but does kind of mess with the `sh.status` integrity a little as we don't really hide fields beyond that. 
I'm unsure how problematic would splitting by version be but that seems to be the only other options (or setting the status to `'unknown'` but that is also a bit weird and would be shown to a lot of users by default

Reply:
> If we went that route, the version would probably be derived from the admin DB right?

The server version? The only reliable version that clients are supposed to make decisions on is the wire version reported by `hello`